src/core/PathUtil.cpp

```diff
@@ -65,51 +103,109 @@ namespace lmms::PathUtil
 		return QDir(baseLocation(base, error));
 	}
 
-	QString basePrefix(const Base base)
+	QString basePrefixQString(const Base base)
+	{
+		const auto prefix = basePrefix(base);
+		return QString::fromLatin1(prefix.data(), prefix.size());
```

Comment:
Why are we going from latin-1 encoding here? Isn't your new `basePrefix` function meant to return a UTF-8 string?